Upstatement/skela-wp-theme
# Fix NPM security issues

## Changes

- Resolves security issues reported by [Dependabot](https://github.com/Upstatement/skela-wp-theme/security/dependabot) via `npm update`

## How To Test

1. Run `npm install`
2. Confirm that `npm run build` completes successfully
3. Confirm that `npm audit` reports no security issues
